app/src/test/java/org/pesmypetcare/mypetcare/controllers/TestTrUpdatePetWash.java

```diff
@@ -50,7 +55,7 @@ public void shouldUpdateWashBody() throws WashAlreadyExistingException {
 
     @Test
     public void shouldUpdateWashDate() throws WashAlreadyExistingException,
-            org.pesmypetcare.usermanagerlib.exceptions.InvalidFormatException {
+        org.pesmypetcare.usermanagerlib.exceptions.InvalidFormatException, ExecutionException, InterruptedException {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary use of fully qualified name 'org.pesmypetcare.usermanagerlib.exceptions.InvalidFormatException' due to existing import 'org.pesmypetcare.usermanagerlib.exceptions.InvalidFormatException'](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5527577)